template/module/static/src/js/module_name.tour.js

```diff
@@ -23,9 +23,13 @@ Tour.register({
             wait: 3000, // Milliseconds to wait
             waitFor: ".start-when-this-jquery-selector-matches-something",
             waitNot: ".start-when-this-jquery-selector-matches-nothing",
-            onend: function() {
-                // Custom code that should return true for step to success
-                return true;
+
+            // Triggers that could return a step title to jump to it
```

Comment:
@yajo thanks a lot!,

Isn't `onload` way more versatile?

I mean, you are documenting great, doc you added must remain but I think we should change the statement for something like: `Executes any JS code,  could return a step title to jump to it.`

Thanks again.